react-native-libraries.json

```
@@ -3306,5 +3306,40 @@
     "android": true,
     "expo": false,
     "npmPkg": "emoji-mart-native"
-  }
+  },
+  {
+    "githubUrl": "https://github.com/infinitered/ignite",
+    "ios": true,
+    "android": true,
+    "expo": false,
+    "npmPkg": "ignite"
+  },
+  {
+    "githubUrl": "https://github.com/infinitered/reactotron",
+    "ios": true,
+    "android": true,
+    "expo": false,
+    "npmPkg": "reactotron"
+  },
+  {
+    "githubUrl": "https://github.com/infinitered/apisauce",
+    "ios": true,
+    "android": true,
+    "expo": false,
+    "npmPkg": "apisauce"
+  },
+  {
+    "githubUrl": "https://github.com/infinitered/gluegun",
+    "ios": true,
+    "android": true,
+    "expo": false,
+    "npmPkg": "gluegun"
+  },
+  {
+    "githubUrl": "https://github.com/infinitered/nsfwjs",
+    "ios": false,
+    "android": false,
+    "expo": false,
+    "npmPkg": "nsfwjs"
+  },
```

```suggestion
  }
```

Remove trailing comma -- this is JSON & it'll break. :)